nfc/libs/NfcCoreLib/lib/Nci/phNciNfc_ListenNfcDep.c

```diff
@@ -45,21 +46,46 @@ phNciNfc_NfcDepLstnRdrAInit(
                 pRemDevInf->tRemoteDevInfo.NfcIP_Info.Nfcip_Active = 0; /* Passive communciation */
             break;
         }
+
         /* Obtain the length of Activation parameters from pBuff */
         ActvnParamsLen = pBuff[7+RfTechSpecParamsLen+PH_NCINFCTYPES_DATA_XCHG_PARAMS_LEN];
```

Comment:
I can guess what you mean but "7+RfTechSpecParamsLen+PH_NCINFCTYPES_DATA_XCHG_PARAMS_LEN != pBuff[7+RfTechSpecParamsLen+PH_NCINFCTYPES_DATA_XCHG_PARAMS_LEN]".
I will use a variable for clarity.